source/more_sampling_tools.Rmd

```diff
@@ -126,96 +131,652 @@ rnd = np.random.default_rng()
 ```
 
 ```{python}
-# The 3 buckets, each representing two trunks on a ship.
-bucket1 = ['Gold', 'Gold']  # First ship.
-bucket2 = ['Gold',  'Silver']  # Second ship.
-bucket3 = ['Silver', 'Silver']  # Third ship.
+# The 3 buckets.  Each bucket represents a ship.  Each has two chests.
+bucket1 = ['Gold', 'Gold']  # Chests in first ship.
+bucket2 = ['Gold',  'Silver']  # Chests in second ship.
+bucket3 = ['Silver', 'Silver']  # Chests in third ship.
 ```
 
 ```{python}
-# Mark trials as not valid to start with.
-# Trials where we don't get a gold trunk first will
-# keep this 'No gold in first trunk' marker.
-second_trunks = np.repeat(['No gold in first trunk'], 10000)
+# For each trial, we will have one of three states:
+#
+# 1. When opening the first chest, it did not contain gold.
+#    We will reject these trials, since they do not match our
+#    experiment description.
+# 2. Gold was found in the first and the second chest.
+# 3. Gold was found in the first, but silver in the second chest.
+#
+# We need a placeholder value for all trials, and will make that
+# "No gold in first chest".
+second_chests = np.repeat(['No gold in first chest'], 10000)
```

Comment:
This discrepancy is still here: "second chest" says "no gold in first chest". That feels like a mismatch.

Reply:
Consider: `never opened`?